libs/cdk/src/lib/template-management/model.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  ChangeDetectorRef,
+  ElementRef,
+  EmbeddedViewRef,
+  NgIterable,
+  NgZone,
+  TemplateRef,
+  ViewContainerRef
+} from '@angular/core';
+import { Observable } from 'rxjs';
+import { StrategyCredentialsMap } from '../render-strategies/model';
+
+export type rxBaseTemplateNames = 'rxError' | 'rxComplete' | 'rxSuspense';
+
+export enum RxBaseTemplateNames {
+  error = 'rxError',
+  complete = 'rxComplete',
+  suspense = 'rxSuspense'
+}
```

Comment:
The naming could map with the input binding of directives? 
`*rxLet="errorTpl: myTpl"`

```typescript
export enum RxBaseTemplateNames {
  error = 'errorTpl',
  complete = 'completeTpl',
  suspense = 'suspenseTpl'
}
```

Reply:
will every directive have it's own template though?

Reply:
Not always imho, but its good to have a 'link' in the names